subsys/logging/log_output.c

```diff
@@ -170,7 +170,7 @@ void log_output_flush(const struct log_output *output)
 }
 
 static int timestamp_print(const struct log_output *output,
-			   uint32_t flags, uint32_t timestamp)
+			   uint32_t flags, log_timestamp_t timestamp)
 {
 	int length;
 	bool format =
```

Comment:
could you add handling of 64 bit timestamp in `!format` case? Currently it always prints `%08lu`

Reply:
formatting case needs more work as it is currently limited to 99 hours.

Reply:
Yes, sure, that's something I can do!
I'll try to make it asap and re-push it again